superset/commands/dashboard/permalink/create.py

```diff
@@ -66,11 +66,21 @@ def __init__(
     def run(self) -> str:
         self.validate()
         dashboard = DashboardDAO.get_by_id_or_slug(self.dashboard_id)
+        print("creating permalink...")
+        
         value = {
             "dashboardId": str(dashboard.uuid),
-            "state": self.state,
+            "state": {**self.state, "urlParams": [['native_filter', '(NATIVE_FILTER-8jS1fx4hl:(extraFormData:(filters:!((col:country_name,op:IN,val:!(Brazil)))),filterState:(label:country_name,validateStatus:!f,value:!(Brazil)),id:NATIVE_FILTER-8jS1fx4hl,ownState:()))']]},
```

Comment:
just need to convert this object to match this schema: 
https://github.com/apache/superset/pull/32196/files#diff-8e89fe05083694b3b8113d73f920a58556865938bbc8119a52562aeced14bfebR248